src/js/keyboardInput.js

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function () {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery");
+
+    gpii.firstDiscovery.charFromKeypress = function (e) {
```

Comment:
It seems that this method is only used by methods under the "gpii.firstDiscovery.keyboardInput" namespace. Maybe it would make sense to move this function under that namespace as well.